url-gen/src/main/kotlin/com/cloudinary/transformation/layer/source/TextSource.kt

```diff
@@ -10,7 +10,7 @@ class TextSource internal constructor(
     private val text: Any,
     private val style: Any,
     private val backgroundColor: Color? = null,
-    private val textColor: Color? = null,
+    private val textColor: Any? = null,
```

Comment:
Does Kotlin support union types?
Can it be `string | Color` instead of Any? 

As for example booleans and objects won't work correctly here.

Reply:
Union types do exist[](https://discuss.kotlinlang.org/t/union-types/77)
In the current design of the SDK we dont use this design pattern but we can start implementing it if required

Reply:
@patrick-tolosa , this is a `private` attribute, not visible for users. It can be `Any`

Reply:
Yes, that is true
Users will only be exposed to the function which allows them to put `Color` or `String`